app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,29 @@
+require 'byebug'
+class MessagesController < ApplicationController
+
+  def index
+      @conversation = Conversation.find_by(id: params[:conversation_id])
+  end
+
+  def show
+    @message = Message.find_by(id: params[:id])
+  end
+
+  def new
+    @conversation = Conversation.find(params[:conversation_id])
+    @message = @conversation.messages.new
+  end
+
+  def create
+    @conversation = Conversation.find(params[:conversation_id])
+    @message = @conversation.messages.new(content: params[:message][:content])
+    @message.user_id = current_user.id
```

Comment:
If you're not using the active record magic getter/setter methods, you might as well save a line and do '@message = @conversation.messages.new(content: params[:message][:content], user_id: current_user.id)'